packages/@o3r/apis-manager/package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "@o3r/apis-manager",
   "version": "0.0.0-placeholder",
-  "description": "APIs manager module of the Otter framework",
+  "description": "This module provides services to help you communicate with your APIs. Its responsibility is to provide an API configuration to a service factory so that it could instantiate an API with the right configurations.\nIt contains a default configuration and a map of specific configurations for API / set of API.\nConfigurations are only exposed through the method `getConfiguration`, which will merge the default configuration and the requested one.",
```

Comment:
```suggestion
  "description": "This module provides services to help you communicate with your APIs. Its responsibility is to provide an API configuration to a service factory so that it could instantiate an API with the right configurations.\nIt contains a default configuration and a map of specific configurations for an API / a set of APIs.\nConfigurations are only exposed through the method `getConfiguration`, which will merge the default configuration and the requested one.",
```